sycl/test-e2e/README.md

```diff
@@ -56,8 +56,8 @@ cmake -G Ninja ...
 ninja check-sycl-e2e
 ```
 
-In addition to this standalone configuration one can enable `check-sycl-e2e`
-target for the sycl-toolchain workspace/build by specifying
+In addition to this, in a non-standalone configuration one can enable
+`check-sycl-e2e` target for the sycl-toolchain workspace/build by specifying
```

Comment:
>  I think it was a mistake in the old readme, but please check my change in case I'm wrong here.

Just a non-native English speaker and probably a missing comma. No ambiguity after you change, so thanks for that.

Reply:
NP! The only thing I was mulling over locally was the use of "non-standalone" vs "in-tree". Do you have any preference? I was actually going to go for "in-tree" but forgot to push.

Reply:
I don't, either is good enough but not perfect :D

Reply:
Fair enough. Since I've had to rebase to try and make some of these CI failures, I've gone with "in-tree" just to help unify the language somewhat.